charts/tfy-karpenter-config/values.yaml

```diff
@@ -252,3 +216,35 @@ karpenter:
     ## @param karpenter.inferentiaProvisionerSpec.providerRefName Name of AWS node template to be used for inferentia provisioner
     ##
     providerRefName: inferentia-default
+
+  ## @section karpenter.defaultNodePoolSpec Node pool spec for default node pool
+  ## 
+  defaultNodePoolSpec:
+    ## @skip karpenter.defaultNodePoolSpec.template
+    template:
+      spec:
+        requirements:
+          - key: karpenter.sh/capacity-type
+            operator: In
+            values: ["spot"]
+          - key: "topology.kubernetes.io/zone"
+            operator: In
+            values: ["eu-west-1a", "eu-west-1b", "eu-west-1c"]
+          - key: kubernetes.io/arch
+            operator: In
+            values: ["amd64"]
+          - key: karpenter.k8s.aws/instance-family
+            operator: NotIn
+            values: ["t3", "t2", "t3a", "p2", "p3", "p4d", "p4de", "p5", "g4dn", "g5", "g4ad", "inf1", "inf2", "trn1", "trn1n"]
+          - key: karpenter.k8s.aws/instance-size
+            operator: NotIn
+            values: ["nano", "micro", "metal"]
+        nodeClassRef:
+          apiVersion: karpenter.k8s.aws/v1beta1
+          kind: EC2NodeClass
+          name: default
+    ## @skip karpenter.defaultNodePoolSpec.disruption
+    disruption:
+      ## @param karpenter.defaultNodePoolSpec.disruption Consolidation policy for default node pool
+      ##
+      consolidationPolicy: WhenUnderutilized
```

Comment:
this should be similar to other node pool specs

Reply:
Why are we not changing values for other node pools, from `gpuProvisionerSpec` to `gpuNodePoolSpec`

Reply:
I am trying to reuse as much of exiting spec where possible to avoid breakage during migration. defaultProvisioner had the complete spec instead of any templates. We can take up the standardisation of the helm chart in another PR, the migration already has too many steps and is going to be error prone